docs/dotcom-v2-migration.md

```diff
@@ -1,125 +1,124 @@
 # Carbon for IBM.com: v2 migration guide
 
 This document will review in detail the changes in Carbon for IBM.com v1 to v2.
-Because Carbon for IBM.com v2 uses Carbon v11, which utilizes Sass modules,
-there is a requirement for teams to use the
-[Dart Sass package](https://sass-lang.com/dart-sass) (`sass`), as opposed to
-`node-sass`. This introduces some new sass concepts such as
-[`@use`](https://sass-lang.com/documentation/at-rules/use) vs
-[`@import`](https://sass-lang.com/documentation/at-rules/import) and
+
+## Global changes
+Carbon for IBM.com v2 uses Carbon v11, which utilizes [Dart Sass](https://sass-lang.com/dart-sass) (`sass`), as opposed to
+`node-sass`. This introduces some new Sass features such as
+[`@use`](https://sass-lang.com/documentation/at-rules/use) and [`@forward`](https://sass-lang.com/documentation/at-rules/forward) keywords ([`@import`](https://sass-lang.com/documentation/at-rules/import) is no longer used), and
 [namespaces](https://sass-lang.com/documentation/at-rules/use#choosing-a-namespace).
 For Carbon v11 migration guidance, see their
 [migration guide](https://github.com/carbon-design-system/carbon/blob/main/docs/migration/v11.md).
 
-## List of component changes
+## Components
 
 | Component                 | Changes                                                                                                                                        |
 | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
 | back-to-top               | No API changes.                                                                                                                                |
 | background-media          | No API changes.                                                                                                                                |
-| button                    | This component is deprecated in v2 in favor for `button-expressive`.                                                                           |
+| button                    | Deprecated in favor of `button-expressive`.                                                                           |
```

Comment:
didn't we end up actually keeping button?

Reply:
I didn't work on this so I'm not 100% sure. But I just spoke with @oliviaflory about it and at the very least it seems like the components are swapped (it should be `button-expressive` deprecated in favor of `button`).

Reply:
ahh yup: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/feat/carbon-for-ibm-dotcom-v2/packages/web-components/src/components/button/button-expressive.ts#L252

Reply:
Ok, got it – `button-expressive` is getting deprecated. Instead, all button usage should come from `@carbon/web-components`.

Reply:
Right.